hudi-flink/src/main/java/org/apache/hudi/streamer/FlinkStreamerConfig.java

```diff
@@ -53,6 +56,10 @@
   @Parameter(names = {"--flink-checkpoint-path"}, description = "Flink checkpoint path.")
   public String flinkCheckPointPath;
 
+  @Parameter(names = {"--flink-state-backend-type"}, description = "Flink state backend type : HashMapStateBackend "
+          + "(default).", converter = FlinkStateBackendConverter.class)
```

Comment:
Flink state backend type, support only hashmap and rocksdb by now, default hashmap